### PR TITLE
[cuutruyen] Update drm script

### PR DESCRIPTION
**Pull request by AccountForStealingCode**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Have not changed source names
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Review by beerpiss — COMMENTED** [review]
`cuuDrmJs` should be split into multiple smaller parts as the extension inspector chokes on super long strings; refer to https://github.com/tachiyomiorg/tachiyomi-extensions/commit/bf058f5dd48e3e9a66754804295f687e621f6773 and the comment left in the interceptor:
https://github.com/tachiyomiorg/tachiyomi-extensions/pull/15398/files#diff-66eac449974c1b9567f883617d58cb3768d6d31ee81ab2251f3dc819f367c584L104

**Comment by beerpiss** [comment]
Also you might want to leave a comment on how `cuuDrmJs` was built, just in case anyone complains

**Comment by AccountForStealingCode** [comment]
That should do the trick.

File: src/vi/cuutruyen/src/eu/kanade/tachiyomi/extension/vi/cuutruyen/CuuTruyenImageInterceptor.kt [hunk 1]
```diff
@@ -86,30 +86,26 @@ class CuuTruyenImageInterceptor : Interceptor {
...
 
     /*
-       rollup -f iife -p terser -n cuudrm cuudrm_bg.js
-       - https://gist.githubusercontent.com/beerpiss/9fe90191d4e67bba1ab69a14618e2036/raw/ce078b70265633f0b619fb6c0d454c0c3c195ab3/cuudrm_bg.js
-       - wasm2js + https://cuutruyen.net/ec6c013a024fb3d1.module.wasm
-
-       Needs to be chunked into smaller strings for dex2jar to work.
```
Comment (arkon) [inline src/vi/cuutruyen/src/eu/kanade/tachiyomi/extension/vi/cuutruyen/CuuTruyenImageInterceptor.kt:104]: Retain this comment.

**Pull request merged**